### ISSUE TITLE
problem with PS docker image

### ISSUE THREAD

**Issue by stashev** [PowerShell/PSReadLine#425]
Just try this image https://hub.docker.com/r/lalyos/powershell/

?1h=PowerShell 
Copyright (C) 2016 Microsoft Corporation. All rights reserved.
nnPS /> nn
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new

Last 0 Keys:
Exception:

System.DivideByZeroException: Attempted to divide by zero.
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)

**Comment by rosanows** [comment]
FWIW,

I get the exception on the microsoft/powershell:latest image, however PS is usable after that one. It happens only once when firing up the docker image.

Here's the output of the script mentioned when opening a new issue:
PS version: 6.0.0-beta                                                                                                                                                                                                                                                          
PSReadline version: 1.2                                                                                                                                                                                                                                                         
os: Linux c27fd96181fd 4.2.8 #1 SMP Tue Jun 6 06:03:33 CST 2017 x86_64 x86_64 x86_64 GNU/Linux                                                                                                                                                                                  
PS file version:

On the docker image there's no powershell.exe, but powershell. $((dir $pshome\powershell).VersionInfo.FileVersion) produces no output for that though.

**Comment by lzybkr** [comment]
Closing - I can't repro using the latest docker images. Reopen if you see it again.
